gcp/table_gcp_compute_disk.go

```diff
@@ -398,3 +397,16 @@ func diskLocation(_ context.Context, d *transform.TransformData) (interface{}, e
 
 	return locationData[param], nil
 }
+
+func getDiskEncryptionKey(_ context.Context, d *transform.TransformData) (interface{}, error) {
```

Comment:
Generally we don't use `get` prefix for transform functions
```suggestion
func getDiskEncryptionKey(_ context.Context, d *transform.TransformData) (interface{}, error) {
```